### DISCUSSION TITLE
Add a link to the newly merged record in the "Merge Succeeded" dialog

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5880] [Feature Requests]
![image](https://github.com/specify/specify7/assets/37256050/b7e40acb-03d5-4494-b74f-e7fd999b1667)

@maxpatiiuk:
>this dialog should have some content besides the header
>we don't have any dialogs without text (with possible exception of loading dialog)

I think a link to the newly merged record would be great just like we see in the notification after it is complete

**Comment by CarolineDenis** [comment]
We would need the newly created agent id in the <Status/> component. Could be included in the response from `status/${mergingId}`
@maxpatiiuk , what do you think?
